java-compiler-testing/src/main/java/io/github/ascopes/jct/containers/impl/AbstractPackageContainerGroup.java

```diff
@@ -128,7 +130,7 @@ public void close() throws IOException {
     for (var container : containers) {
       try {
         container.close();
-      } catch (IOException ex) {
+      } catch (Exception ex) {
```

Comment:
- [ ] Should this only catch IOException now that the superinterface uses Closeable rather than AutoCloseable?